bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableTable.java

```diff
@@ -86,25 +85,16 @@
   public AbstractBigtableTable table;
 
   @Before
-  public void setup() {
-    BigtableOptions options =
-        BigtableOptions.builder()
-            .setAdminHost("localhost")
-            .setDataHost("localhost")
-            .setPort(0)
-            .setProjectId(TEST_PROJECT)
-            .setInstanceId(TEST_INSTANCE)
-            .setRetryOptions(RetryOptions.builder().setEnableRetries(false).build())
-            .setCredentialOptions(null)
-            .setUserAgent("testAgent")
-            .build();
-
```

Comment:
Shouldn't these values be ported over?  ie
config.set(BIGTABLE_ADMIN_HOST_KEY, "localhost")